index.html

```diff
@@ -49,5 +49,191 @@ <h1 id="name">
+                    </ul>
+                    <p class="work-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="work-tools">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+        </div>
+        <div class="portfolio">
+            <div class="project">
+                <div class="work-img">
+                    <img src="/assets/images/projects/tonic.png" alt="tonic image">
+                </div>
+                <div class="work-description">
+                    <h2 class="work-title">tonic</h2>
+                    <ul class="canopy">
+                        <li>canopy</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="work-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="work-tools">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+        </div>
+        <div class="portfolio">
+            <div class="project">
+                <div class="work-img">
+                    <img src="/assets/images/projects/anoter-one.png" alt="tonic image">
+                </div>
+                <div class="work-description">
+                    <h2 class="work-title">Multi-Post Stories</h2>
+                    <ul class="canopy">
+                        <li>canopy</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="work-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="work-tools">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section id="about">
+        <div class="about-container">
+            <h1 class="name">About Myself</h1>
+            <p class="details">
+                Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
```

Comment:
- [ ] There are few issues in this section. kindly consider addressing them
- [ ] We should not use `h1` tag after taking one h1 tag for the main heading in the hero section. the reason is to follow the best semantics and best practices, so consider avoiding that.
- [ ] The `About Myself title has 40px font-size and 700 font-weight in the figma design, so consider implementing that.
- [ ] The <p> element after has 400 weight in the figma design but in your design it looks stronger that that. so consider fixing it.
your design
![as](https://user-images.githubusercontent.com/96683843/234930974-562c8d12-1564-4fc7-84e1-38a592b831af.jpg)

figma design
![fd](https://user-images.githubusercontent.com/96683843/234930961-c9e8e97d-c882-403d-9f77-7eee1ffbb52e.jpg)